### PR TITLE
Add Swagger

### PR DESCRIPTION
**Pull request by gregfrasco**
Since `/docs` is not working how about Swagger

### CODE REVIEW

**Comment by tavousi** [comment]
Swagger is awsome ! thanks @gregfrasco

**Comment by richardzyx** [comment]
#176 Here's the discussion from my previous PR on adding swagger.

File: manifest.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 const Confidence = require('confidence');
 const Config = require('./config');
 const Path = require('path');
-
+const Package = require('./package.json');
```
Comment (jedireza) [inline manifest.js]: Please group all `require`s alphabetically and leave two empty lines after the last one.

File: manifest.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 'use strict';
 const Confidence = require('confidence');
 const Config = require('./config');
+const Package = require('./package.json');
 const Path = require('path');
 
-
```
Comment (jedireza) [inline manifest.js]: And, don't remove this empty line.

**Pull request merged**